docs/userguide/ui-concepts.md

```diff
@@ -158,15 +32,75 @@ Your `Screen` implementations can be defined completely separately from their vi
     "The Cart screen is shown in a modal over the Home screen on tablets."
     It's a safe bet you understood each of those sentences.
 
-## Composition and Navigation: Screens all the way down
+## Workflow Tree, Rendering Tree, View Tree
+
+In the Workflow Core page we discussed how Workflows can be [composed as trees](../concepts#composing-workflows), like this email app driven by a trio of Workflows that assemble a composite `SplitScreen` rendering.
+
+![Workflow schematic showing a parent EmailBrowserWorkflow assembling the renderings of its children, InboxWorkflow and MessageWorkflow, into a SplitScreen(InboxScreen, MessageScreen)](../images/email_schematic_renderings_only.svg)
+
+Let's take a look at how Workflow UI transforms such a [container screen](../../glossary#container-screen) into a [container view](../../glossary#container-view).
+
+![A box labeled Runtime contains the EmailBrowserWorkflow. It slightly overlaps a larger box labeled Native view system. A line from the EmailBrowserWorkflow's Rendering port connects to a  box at the top of the Native View System, labeled Workflow root container. That Rendering, a SplitScreen(InboxScreen, MessageScreen), is passed from the Workflow root container down to a bi-part box labeled Workflow container / Custom split view. From there, InboxScreen is passed down to a similar bi-part box labeled Workflow container / Custom inbox view, and MessageScreen to Workflow container / Custom message view](../images/down_the_view_tree.svg)
+
+The main connection between the Workflow Core runtime and a native view system is the stream of Rendering objects from the root Workflow, `EmailBrowserWorkflow` in this illustration.
+From that point on, the flow of control is entirely in view-land.
+
+The precise details of that journey vary between Android and iOS in terms of naming, subclassing v. delegating, and so on, mainly to ensure that the API is idiomatic for each audience.
+None the less, the broad strokes are the same.
+(Move on to [Coding Workflow UI](../ui-in-code) to drill into the platform-specific details.)
+
+Each flavor of Workflow UI provides two core container helpers, both pictured above:
+
+* A "workflow container", able to instantiate and update a view that can display Screen instances of the given type
+    * In iOS this is `DescribedViewController`
+    * For Android Classic we provide `WorkflowViewStub`, very similar to `android.view.ViewStub`.
+    * Android Jetpack Compose code can call `@Compose fun WorkflowRendering()`.
+* A "workflow root container", able to field a stream of renderings from the Workflow Core runtime, and pass them on to a workflow container
+    * `ContainerViewController` for iOS
+    * `WorkflowLayout` for Android Classic
+    * `@Compose fun Workflow.renderAsState()` for Android Jetpack Compose
+
+When the runtime in our example is started, the flow is something like this:
```

Comment:
It's only 20 lines above. Should we move it here?

Reply:
I've moved it below line 50, which now says "Each flavor of Workflow UI provides two core container helpers, both pictured below" ("below" instead of "above"). PTAL